python/pylibvw.cc

```diff
@@ -76,6 +81,12 @@ void my_save(vw_ptr all, std::string name)
 { VW::save_predictor(*all, name);
 }
 
+py_cpp_bridge_ptr get_python_cpp_bridge_ptr(vw_ptr all)
+{ 
+  RED_PYTHON::PythonCppBridge* temp = (RED_PYTHON::PythonCppBridge*)(all->pythonCppBridge);
```

Comment:
check if all->pythonCppBridge is null in case user calls in a vw without --red-python arg